README.md

```diff
@@ -1,16 +1,123 @@
-# Title
+# Method Scope Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Define a local variable with the expected value
+- Recognize the restrictions of local variable scope
```

Comment:
This is one of those LGs that is hard to measure. How could we measure "recognition?" I think this should be cut.